examples/README.md

```diff
@@ -1,7 +1,10 @@
 `waveorder` is undergoing a significant refactor, and this `examples/` folder serves as a good place to understand the current state of the repository.
 
-The `models/` folder demonstrates the latest functionality of `waveorder`. These scripts will run as is in an environment with `waveorder` and `napari` installed. Each script demonstrates a simulation and reconstruction with a **model**: a specific set of assumptions about the sample and the data being acquired. 
+Some examples require `napari`. Start with `pip install napari[all]`, and visit the [napari installation guide](https://napari.org/dev/tutorials/fundamentals/installation.html) if that fails.  
 
-The `maintenance/` folder demonstrates the functionality of `waveorder` that we plan to move to `models/`. These scripts can be run as is, and they are being maintained with tests.
-
-The `documentation/` folder consists of examples that demonstrate reconstruction with real data. These examples require access to the complete datasets, so they are not being actively maintained and serve primarily as documentation.
\ No newline at end of file
+| Folder      | Requires                   | Description                                                                                           |
+|------------------|----------------------------|-------------------------------------------------------------------------------------------------------|
+| `models/`        | `waveorder`, `napari`      | Demonstrates the latest functionality of `waveorder` through simulations and reconstructions using various models. |
+| `maintenance/`   | `waveorder`                | Showcases functionality planned for `models/` folder; scripts are maintained with automated tests.               |
```

Comment:
Do you mean, "Examples of computational imaging methods enabled by functionality of `waveorder`; these scripts are maintained with automated tests"

Reply:
That's correct...fixed. Thanks.